src/components/interfaces/HMI_API.xml

```diff
@@ -1432,16 +1432,25 @@
 <enum name="CharacterSet">
   <description>The list of potential character sets</description>
   <element name="TYPE2SET">
-    <description>See [@TODO: create file ref]</description>
+    <description>See [@TODO: create file ref]. Note that this element is deprecated in the MOBILE API</description>
```

Comment:
same question from MOBILE_API, why keep `See [@TODO: create file ref]. `

Reply:
Removed in https://github.com/smartdevicelink/sdl_core/pull/3454/commits/e630938b2b5a012d0c9f8cb9fa4f73a7e7d4a198